build/release.mk

```diff
@@ -108,6 +108,9 @@ else
 endif
 	cp NOTICE.txt $(DIST_PATH)
 	cp README.md $(DIST_PATH)
+	if [ -f manifest.txt ]; then \
+		cp manifest.txt $(DIST_PATH); \
+	fi; \
```

Comment:
`; \` at the end of the line should be removed.

Reply:
Which line? The whole file has a semicolon at the end though. It would keep it consistent.

Reply:
only line 113, since it's the last line of multi-line if block. `;` does nothing special here but `\` is going to try to append the next line to it.